pkg/skaffold/runner/runner.go

```diff
@@ -170,6 +172,19 @@ func getTester(cfg []*latest.TestCase, opts *config.SkaffoldOptions) (test.Teste
 	}
 }
 
+// getContext returns the kubectl context to use for deployment
+func getContext(originalContext, schemaContext string) string {
```

Comment:
This logic should be moved and merged into cmd/skaffold/app/cmd/config/util.go:36